.chloggen/metric-span-component-attributes.yaml

```
@@ -0,0 +1,37 @@
+# Use this changelog template to create an entry for release notes.
+
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: 'enhancement'
+
+# The name of the component, or a single word describing the area of concern, (e.g. otlpreceiver)
+component: service
+
+# A brief description of the change.  Surround your text with quotes ("") if it needs to start with a backtick (`).
+note: Internal telemetry uses scope attributes to identify the source component
+
+# One or more tracking issues or pull requests related to the change
+issues: [12217]
+
+# (Optional) One or more lines of additional information to render under the primary note.
+# These lines will be padded with 2 spaces and then inserted directly into the document.
+# Use pipe (|) for multiline entries.
+subtext: |
+  All internal telemetry now includes one or more of the following instrumentation scope attributes:
+  - `otelcol.component.kind`
+  - `otelcol.component.id`
+  - `otelcol.pipeline.id`
+  - `otelcol.signal`
+  - `otelcol.signal.output`
+  These attributes are defined in the [Pipeline Component Telemetry RFC](https://github.com/open-telemetry/opentelemetry-collector/blob/main/docs/rfcs/component-universal-telemetry.md#attributes),
+  and identify the component instance from which the telemetry originates.
+  These attributes are added automatically without changes to component code by wrapping the
+  `MeterProvider`, `TracerProvider`, and `Logger` provided to them.
+  Components will be able to opt out of one or more of these attributes, but that API is not yet
+  publicly available.
```

I would remove the implementation details and forward-looking statements
```suggestion
  These attributes are added automatically without changes to component code.
```